src/components/VmDetails/BaseCard.js

```diff
@@ -160,8 +164,9 @@ BaseCard.propTypes = {
   editMode: PropTypes.bool,
   editable: PropTypes.bool,
   disableSaveButton: PropTypes.bool,
-  editTooltip: PropTypes.string,
-  disableTooltip: PropTypes.string,
+  editTooltip: tooltipPropType,
```

Comment:
After other changes:
```suggestion
  editTooltip: Tooltip.propTypes.tooltip,
```